variables.go

```diff
@@ -43,12 +54,13 @@ func (vs Vars) toStringMap() (m map[string]string) {
 	return
 }
 
-var (
-	// ErrCantUnmarshalVar is returned for invalid var YAML
-	ErrCantUnmarshalVar = errors.New("task: can't unmarshal var value")
-)
+// Var represents either a static or dynamic variable.
+type Var struct {
+	Static string
+	Sh     string
+}
 
-// UnmarshalYAML implements yaml.Unmarshaler interface
+// UnmarshalYAML implements yaml.Unmarshaler interface.
```

Comment:
The trailing period is recommended by [Effective Go](https://golang.org/doc/effective_go.html), or at least the way I interpret it:

> Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared.

The standard lib uses a period consistently throughout all libraries as far as I have seen. Note that I am only talking about doc comments, not code comments.